tensorflow/compiler/mlir/hlo/lib/Dialect/mhlo/transforms/lhlo_legalize_to_affine.cc

```diff
@@ -98,6 +98,66 @@ struct DotOpConverter : public OpRewritePattern<DotOp> {
   }
 };
 
+struct ConcatOpConverter : public OpRewritePattern<ConcatenateOp> {
```

Comment:
Can you document the pattern here?

Reply:
Added, Please review, If you feel I am missing anything.